content/about.md

```diff
@@ -0,0 +1,11 @@
+---
+index: 4
+name: Peter Doolaard
```

Comment:
Twee keer Peter? En twee keer index 4? Welke klopt niet?

Reply:
fixed and sorted